secator/query/json.py

```
@@ -194,3 +194,44 @@ def _execute_update(self, query: dict, update: dict) -> int:
+			try:
+				path_info = get_info_from_report_path(path)
+				with open(path, 'r') as f:
+					data = _json.load(f)
+				info = data.get('info', {})
+				info['_type'] = path_info.get('type', '')
+				info['_id'] = path_info.get('type', '') + '/' + path_info.get('id', '')
+				info['_workspace'] = path_info.get('workspace', '')
+				info['_path'] = str(path)
+				runners.append(info)
+			except Exception:
+				continue
```

Ruff (0.15.15)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)